apps/ai-content-generator/src/components/app/dialog/common-generator/field-selector/content-source/ContentSource.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, FormControl, Select } from '@contentful/f36-components';
+
+interface Props {
+  title: string;
+  isNewText: boolean;
+  onChange: () => void;
+  selectFieldWidth: number;
+}
+
+function EntryFieldList(props: Props) {
```

Comment:
Looks like this could be renamed to `ContentSource`

Reply:
Good catch, thank you!